### DISCUSSION TITLE
Onboarding call

### DISCUSSION THREAD

**Discussion by kirillrogovoy** [org-roam/org-roam-ui#4] [Product]
If you want an onboarding video call with the core team, please leave a comment here with some contact details. We speak English, Dutch, Russian, Ukrainian.

**Comment by jhhb** [comment]
Hello,

Thanks for the awesome project! I came here from the "Contributing" section of the README and have a [small patch against my fork that I wanted to contribute](https://github.com/jhhb/org-roam-ui/pull/1) to deal with #17 as I ran into it while starting to use org-roam-ui.

If an onboarding call is the preferred way to start contributing, I'm available by email at: `fakeanon (at) fastmail.com` for further conversation, otherwise, I'm happy to discuss further on #17 directly.

Thanks for your time!

**Reply by orusmateo** [reply]
Hey team! Firstly thank you for this amazing contribution. 

I’m working with OCADU and Harvard on the impacts of 4D qualitative data modeling. I would appreciate a friendly checkin over video call! 

Looking forward to your response!

**Reply by tefkah** [reply]
hi @orusmateo! That could be arranged, what would be your goal for the call? Do you want to contribute to org-roam-ui, or would you like to use it someway related to your work?

**Comment by orusmateo** [comment]
Hello Thomas,

 I would like both to contribute to the visioning of org-roam-ui, and to test it as a case study with data sets I can provide.


How can I book with you? Perhaps I can refer to some sort of booking calendar since I am in EST.


Thank you,

Orus Mateo
________________________________

**Reply by tefkah** [reply]
you can book a time with me!

**Comment by venikx** [comment]
I’m kinda jealous of what you can do with obsidian now, and I don’t like to admit it, but I’ve tried switching many time already (org-mode is just too good).

I’d like to focus some effort on getting some work done regarding publishing your org-roam-ui either using ssr or ssg. I’m relatively familiar with Next.js and React. 

Would someone be able to give me a rundown of the project? I’ve already been tinkering with it on my own, but I think having an overview of a person who worked in the codebase might be cool (specifically focusing on some of the org and emacs specifics).

Interesting concept btw! :)
I speak Dutch and English, if I’m a bit drunk you might catch me speaking what I think is French ;)

**Reply by tefkah** [reply]
oo very glad to hear you want to contribute!

I'm also kind of jealous of obsidian, would love to work with you to help us be a little less so :)

you can book an appointment with me here so i can give you the lay of the land: https://cal.com/tefkah

i've been wanting to work on publishing for a while, but never got around to it properly. i've been working on doing so in a slightly different fashion [here](https://github.com/tefkah/zkp), but trying to keep everything static proved to be a royal pain in the ass, i'm looking forward to hearing your perspective on this!
